src/commands/removecolumncommand.js

```diff
@@ -56,41 +52,74 @@ export default class RemoveColumnCommand extends Command {
 		const tableMap = [ ...new TableWalker( table ) ];
 
 		// Get column index of removed column.
-		const cellData = tableMap.find( value => value.cell === tableCell );
-		const removedColumn = cellData.column;
-		const selectionRow = cellData.row;
-		const cellToFocus = getCellToFocus( tableCell );
+		const firstCellData = tableMap.find( value => value.cell === firstCell );
+		const selectionRow = firstCellData.row;
+		const cellsToFocus = getCellToFocus( firstCell, lastCell );
+
+		const removedColumnIndexes = {
+			first: firstCellData.column,
+			last: tableMap.find( value => value.cell === lastCell ).column
+		};
 
 		model.change( writer => {
+			// A temporary workaround to avoid the "model-selection-range-intersects" error.
+			writer.setSelection( writer.createSelection( table, 'on' ) );
```

Comment:
\`setSelection( createRangeOn( table ) )\`

Reply:
Fixed outside of the GH.